tests/poll.rs

```diff
@@ -112,6 +112,7 @@ fn poll_closes_fd() {
 }
 
 #[test]
+#[cfg_attr(target_os = "hurd", ignore = "protocol not supported")]
```

Comment:
Does Hurd not support TCP? Can you expand on what protocol is not supported?

Reply:
TCP is supported of course :) I have to check again precisely, it's probably some particular sockopt, possibly the failure could be ignored inside the mio implementation.

Reply:
It's the `set_read_timeout` call that gets `SO_RCVTIMEO` seen as not being implemented. I have disabled them, it then goes fine.